src/index.ts

```diff
@@ -161,32 +175,26 @@ export function transpose(
 	return commandsToDockerfile(outCommands);
 }
 
-// FIXME: This is taken from resin-io-modules/resin-bundle-resolve
-// export this code to a shared module and import it in this project
-// and resin-bundle-resolve
-export function normalizeTarEntry(name: string): string {
-	const normalized = path.normalize(name);
-	if (path.isAbsolute(normalized)) {
-		return normalized.substr(normalized.indexOf('/') + 1);
-	}
-	return normalized;
-}
-
 const getTarEntryHandler = (
 	pack: tar.Pack,
 	dockerfileName: string,
 	opts: TransposeOptions,
 ) => {
+	const streamToPromise = require('stream-to-promise');
 	return (
-		header: tar.TarHeader,
+		header: tar.Headers,
```

Comment:
The previous `tar.TarHeader` type was coming from this repo's `typings/tar-stream.d.ts`, which I've deleted and replaced with `"@types/tar-stream": "~1.6.1"` in `package.json`.